homeassistant/package_constraints.txt

```diff
@@ -2,6 +2,7 @@ aiohttp==3.3.2
 astral==1.6.1
 async_timeout==3.0.0
 attrs==18.1.0
+bcrypt==3.1.4
```

Comment:
So one downside of using bcrypt is that it requires an external dependency. They are providing wheels for Windows, Mac and Linux, so that's good:

![image](https://user-images.githubusercontent.com/1444314/44328613-47041400-a462-11e8-9f46-57152433d8d4.png)

For Python 3.7 there are currently only wheels for Windows. Windows users have the most difficulty building stuff, so that's good. 

Because it's part of the PYCA org, I think I would trust it to remain maintained.